PlayerController.cs

```diff
@@ -9,7 +9,13 @@ public class PlayerController : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
-        
+        // store the user's name
+        string username;
+        // name the username "xidodan"
+        username = "xiaodan";
+
+        //print thia message
+        Console.WriteLine("My player's name is " + username);
```

Comment:
Not quite!! We want to print to Unity's console. Here, we can use Unity's [Debug.Log](https://docs.unity3d.com/ScriptReference/Debug.Log.html) function. So the correct code should be: 
`Debug.Log( "My player's name is " + username);`